### DISCUSSION TITLE
Question regarding CameraCalibration

### DISCUSSION THREAD

**Discussion by johannes-do** [foxglove/community#925] [Q&A] [answered]
I am a bit confused about the [CameraCalibration ](https://docs.foxglove.dev/docs/visualization/message-schemas/camera-calibration) schema. 

The projection matrix should be the intrinsics_matrix * extrinsics_matrix. But it seems for translation and rotation from one frame to another the FrameTransform is used (which makes sense). Thus the projection matrix of the camera only contains the intrinsic. Then what is the difference between the P and the K? When would these two be different?

**Comment by jtbandes** [comment, marked as answer]
These would be different for stereo cameras. However the app does not support stereo cameras right now, and so we are planning to update the code to use only the `K` matrix (not `P`) in the future.
